README.md

```diff
@@ -177,8 +172,35 @@ while True:
 c.close()
 ```
 
-See the [examples](examples) directory for more examples, including [how to configure](examples/confluent_cloud.py) the python client for use with
-[Confluent Cloud](https://www.confluent.io/confluent-cloud/).
+**AdminClient**
```

Comment:
Missing ":" at end of "AdminClient", or remove from "Producer:" and "Consumer:"
while you're editing the file, the Avro example headers are missing a ":"

Reply:
these are all effectively headings, so all better off without the ":" imo, removed everywhere.